UPGRADE-6.0.md

```diff
@@ -43,6 +43,11 @@ Messenger
  * Removed RedisExt transport. Run `composer require symfony/redis-messenger` to keep the transport in your application.
  * Use of invalid options in Redis and AMQP connections now throws an error.
 
+PHPUnitBridge
+-------------
+
+ * Removed support for `@expectedDeprecation` annotaiton. Use `expectDeprecation()` method instead.
```

Comment:
Removed support for `@expectedDeprecation` annotations, use the `ExpectDeprecationTrait::expectDeprecation()` method instead.